app/controllers/recommendations_controller.rb

```diff
@@ -1,7 +1,15 @@
 # frozen_string_literal: true
 
 class RecommendationsController < AuthenticationController
-  before_action :set_planning_application
+  before_action :set_planning_application, only: %i[new create]
```

Comment:
Oo...I think it wouldn't break anything, and would actually be more efficient, to use `set_planning_application_with_recommendations` instead of `set_planning_application` for new and create!

Reply:
Didn't think any recommendation records were loaded in this action. Let me double check